installer/bundle/bundle_skel.sh

```diff
@@ -387,6 +411,58 @@ install_if_program_does_not_exist_on_system()
     fi
 }
 
+install_troubleshooter()
```

Comment:
I'd second Henry's point below, t-shoot tool installation should not add a failure mode for OMS agent installation or cleanup/purge. Can you please d-check or add a setup test (if we have such infra at all at the moment) that failure in t-shoot setup does not prevent OMS setup and that having t-shoot tool installed does not prevent OMS uninstall or leaves leftovers?

Reply:
I changed it so `install_troubleshooter` isn't called until the end of OMS install (in the `cleanup_and_exit` script, so as to limit the possible interference), as well as some other assurances that it won't affect OMS installation / cleanup even if it fails (see more details at https://github.com/microsoft/OMS-Agent-for-Linux/pull/1190#issuecomment-663785862 )